page/download.summary.njk

```diff
@@ -0,0 +1,554 @@
+{%- set assetUrl = assetUrl | default(assetPath) -%}
```

Comment:
is this a "magic" variable used elsewhere? can't see it being used in this file

Reply:
Not sure what this is, I will address this when it comes to the next story which addresses the actual HTML.